resources/assets/js/app/controllers/IdeaHubController.js

```diff
@@ -64,6 +68,32 @@ module.exports = Marionette.Object.extend({
         app.render(view);
 
     },
+    createVote: function () {
+        var VoteAnswers = new VoteAICollection([{}, {}], {parentUrl: ''});
+        var UsersCollection = new usersCollection();
+        var accessedUsers = new usersCollection();
+
+        UsersCollection.fetch();
+
+        UsersCollection.opposite = accessedUsers;
+        UsersCollection.glyph = 'plus';
+        accessedUsers.opposite = UsersCollection;
+        accessedUsers.glyph = 'minus';
+
+        var model = new VoteModel();
+        var view = new CreateVote({
+            model: model,
+            answers: VoteAnswers,
+            users: UsersCollection,
+            accessedUsers: accessedUsers
+        });
+
+        view.listenTo(Radio.channel('votesChannel'), 'createEmptyVoteItem', function (col) {
+            col.add(new VoteAImodel({}, {parentUrl: col.parentUrl}));
```

Comment:
you no need to set parentUrl per each new model, if model doesn't have parentUrl it will be taken from collection